elements/modal.js

```diff
@@ -146,7 +146,10 @@ function confirmModal () {
 
     var deleteButton = button.green('Yes, Remove Dat', {
       class: 'fr ml3 confirm-button',
-      onclick: ondelete
+      onclick: ondelete,
+      onload: function () {
```

Comment:
I believe the first argument passed in is a reference to `el` so you could do:
```js
function (el) {
  el.focus()
}
```
(if I'm not mistaken)